SPIFlashLogger.class.nut

```diff
@@ -224,6 +224,68 @@ class SPIFlashLogger {
         });
     }
 
+    function readSync(index = 1) {
```

Comment:
It should be a useful function for public use except that it only reads one object, the first or last. So it might as well be private. (_readSync).